packages/sdk/test/job.js

```diff
@@ -0,0 +1,41 @@
+const path = require('path');
+const { readJson } = require('fs-extra');
```

Comment:
Ditto, if we have no ability to run in the browser, we need to add FIXME comment, either.

Reply:
The api `job.run` and `pipeline.install` changed the argument to object from filename, it should work in browser now.

Reply:
What I meant is that fs-extra is not working at the browser, right?

Reply:
Yes, but it's just used in test. I think no need to fix.